app/routes.py

```diff
@@ -1,2 +1,49 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, abort, make_response
 
+class Planets:
```

Comment:
It's a standard convention across programming languages to use singular names for class names, so I recommend changing this to `class Planet:`